### PR TITLE
Add blogpost for 0.12.0-RC1 and 0.11.0

### PR DESCRIPTION
**Pull request by biboudis**

### CODE REVIEW

File: docs/blog/_posts/2019-01-17-12th-dotty-milestone-release.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+<!--more-->
+
+This is our 12th scheduled release according to our
+[6-week release schedule](https://dotty.epfl.ch/docs/usage/version-numbers.html).
+
+## What’s new in the 0.12.0-RC1 technology preview?
+
+### Extension Methods
+
+We are excited to announce that extension methods are now offered through dedicated language support!
```
Comment (smarter) [inline docs/blog/_posts/2019-01-17-12th-dotty-milestone-release.md]: I think it's important to note that:
- Extension methods have not gone through the SIP process yet
- In particular, the syntax might still change (and link to the PR where most of the discussion took place so far)

File: docs/blog/_posts/2019-01-17-12th-dotty-milestone-release.md [hunk 2]
```diff
@@ -61,8 +61,8 @@ The decompiler can be invoked with the corresponding flag: `dotc -decompile xyz.
 
 On the programmatic side of TASTy we are rolling out changes according to our plan discussed at [Macros: The Plan for Scala 3](https://www.scala-lang.org/blog/2018/04/30/in-a-nutshell.html). 
 In this release, we make progress following the _Next Steps_ of the aforementioned blogpost by offering constructors that work directly with reflect trees. 
-Consequently, TASTy extractors meet their dual, TASTy constructors.
-We also connect the new lower-level reflection layer to the existing principled macro system based on quotes and splices offerring two new expression methods for `Expr[T]`:
+Consequently, TASTy extractors meet their dual, TASTy constructors!
+We also connect the new lower-level reflection layer to the existing principled macro system based on quotes and splices offerring, two new expression methods for `Expr[T]`:
```
Comment (SethTisue) [inline docs/blog/_posts/2019-01-17-12th-dotty-milestone-release.md]: offerring -> offering

Comment (biboudis) [reply docs/blog/_posts/2019-01-17-12th-dotty-milestone-release.md]: Oops. Thx!

File: docs/blog/_posts/2019-01-17-12th-dotty-milestone-release.md (hunk 1 above)
Comment (biboudis) [reply docs/blog/_posts/2019-01-17-12th-dotty-milestone-release.md]: Resolving since everything will go through the SIP process eventually. Thx!

**Comment by LPTK** [comment]
> Previously, one had to place the desired method in the trait with the receiver object to extend, as part of the parameter list. The infix-ness of the method was being restored by manually writing the implicit class, resolving `implicitly` that extended object.

This paragraph is extremely obscure. I believe what you meant to say was just:

> Previously, one had to place the desired methods in an implicit class.

But that doesn't even mention why Dotty extension methods are a big deal and make type classes easier (they use the implicit scope).

Would a PR be welcome to change that?

**Comment by smarter** [comment]
Yes.

**Comment by biboudis** [comment]
Yes, of course! 

If possible rephrase the sentences above still indicating the verbosity of the previous approach while adding the missing bit of info about the implicit instance enabling the extension method(s).

Thanks!

**Pull request merged**